app/src/main/java/com/github/multimatum_team/multimatum/activity/DeadlineDetailsActivity.kt

```diff
@@ -228,6 +238,18 @@ class DeadlineDetailsActivity : AppCompatActivity() {
 
     }
 
+    // Change the color of the date and title views according to the theme
+    private fun adaptToCurrentTheme() {
+        val isNightMode = sharedPreferences.getBoolean(MainSettingsActivity.DARK_MODE_PREF_KEY, false)
```

Comment:
This means that if no value is found in `sharedPreferences` then dark mode is disabled right? I think that ideally the `false` default value should be a constant, named e.g. `IS_DEFAULT_DARK_MODE_ENABLED`.

Reply:
Good point